### PR TITLE
Standardization part 2

### PR DESCRIPTION
**Pull request by bmwill**
Per #1119 continue standardizing our bin, argument and crate names

### CODE REVIEW

File: benchmark/Cargo.toml [hunk 1]
```diff
@@ -6,24 +6,23 @@ publish = false
...
 [dependencies]
-clap = "2.33.0"
 futures = "0.1.28"
 grpcio = { version = "0.4.4", default-features = false, features = ["protobuf-codec"] }
 lazy_static = "1.2.0"
 protobuf = "~2.7"
 rand = "0.6.5"
 regex = { version = "1.3.0", default-features = false, features = ["std", "perf"] }
-structopt = "0.2.15"
+structopt = "0.3.2"
```
Comment (huitseeker) [inline benchmark/Cargo.toml]: <3 finding these details here.

File: benchmark/src/cli_opt.rs [hunk 2]
```diff
@@ -21,69 +20,68 @@ arg_enum! {
...
     pub metrics_server_address: Option<String>,
     /// Valid faucet key file path.
-    #[structopt(short = "f", long = "faucet_key_file_path", required = true)]
+    #[structopt(short = "f", long, required = true)]
     pub faucet_key_file_path: String,
     /// Number of AC clients.
     /// If not specified or equals 0, it will be set to #validators * #cpus. Why this value?
     /// We want to create/connect 1 AC client to each AC thread on each validator.
-    #[structopt(short = "c", long = "num_clients", default_value = "0")]
+    #[structopt(short = "c", long, default_value = "0")]
```
Comment (huitseeker) [inline benchmark/src/cli_opt.rs]: Nice catches for the redundant field names — TIL.

File: libra-swarm/src/main.rs [hunk 3]
```diff
@@ -8,36 +8,32 @@ use structopt::StructOpt;
...
+#[structopt(about = "Libra swarm to start local nodes")]
 struct Args {
     /// Number of nodes to start (1 by default)
-    #[structopt(short = "n", long = "num_nodes")]
-    pub num_nodes: Option<usize>,
+    #[structopt(short = "n", long, default_value = "1")]
+    pub num_nodes: usize,
     /// Enable logging
-    #[structopt(short = "l", long = "enable_logging")]
+    #[structopt(short = "l", long)]
```
Comment (huitseeker) [inline libra-swarm/src/main.rs]: Would comments on the default behavior make sense here?

Comment (bmwill) [reply libra-swarm/src/main.rs]: yep will do

**Pull request closed without merging**